website/blog/authors.yml

```diff
@@ -1,125 +1,125 @@
-  image_url: https://github.com/LeoLjl/leoljl.github.io/blob/main/profile.jpg?raw=true
-
-thinkall:
-  name: Li Jiang
-  title: Senior Software Engineer at Microsoft
-  url: https://github.com/thinkall
-  image_url: https://github.com/thinkall.png
-
-rickyloynd-microsoft:
-  name: Ricky Loynd
-  title: Senior Research Engineer at Microsoft
-  url: https://github.com/rickyloynd-microsoft
-  image_url: https://github.com/rickyloynd-microsoft.png
-
-samershi:
-  name: Saleema Amershi
-  title: Senior Principal Research Manager at Microsoft Research
-  url: https://github.com/samershi
-  image_url: https://github.com/samershi.png
-
-pcdeadeasy:
-  name: Piali Choudhury
-  title: Principal RSDE at Microsoft Research
-  url: https://github.com/pcdeadeasy
-  image_url: https://github.com/pcdeadeasy.png
-
-victordibia:
-  name: Victor Dibia
-  title: Principal RSDE at Microsoft Research
-  url: https://github.com/victordibia
-  image_url: https://github.com/victordibia.png
-
-afourney:
-  name: Adam Fourney
-  title: Principal Researcher Microsoft Research
-  url: https://www.adamfourney.com
-  image_url: https://github.com/afourney.png
-
-beibinli:
-  name: Beibin Li
-  title: Senior Research Engineer at Microsoft
-  url: https://github.com/beibinli
-  image_url: https://github.com/beibinli.png
-
-gagb:
-  name: Gagan Bansal
-  title: Senior Researcher at Microsoft Research
-  url: https://www.linkedin.com/in/gagan-bansal/
-  image_url: https://github.com/gagb.png
-
-jieyuz2:
-  name: Jieyu Zhang
-  title: PhD student at University of Washington
-  url: https://jieyuz2.github.io/
-  image_url: https://github.com/jieyuz2.png
-
-julianakiseleva:
-   name: Julia Kiseleva
-   title: Senior Researcher at Microsoft Research
-   url: https://github.com/julianakiseleva/
-   image_url: https://avatars.githubusercontent.com/u/5908392?v=4
-
-narabzad:
-   name: Negar Arabzadeh
-   title: PhD student at the University of Waterloo
-   url: https://www.negara.me/
-   image_url: https://github.com/Narabzad.png
-
-LinxinS97:
-  name: Linxin Song
-  title: MS student at Waseda University
-  url: https://linxins97.github.io/
-  image_url: https://github.com/LinxinS97.png
-
-skzhang1:
-  name: Shaokun Zhang
-  title: PhD student at the Pennsylvania State University
-  url: https://github.com/skzhang1
-  image_url: https://github.com/skzhang1.png
-
-olgavrou:
-  name: Olga Vrousgou
-  title: Senior Software Engineer at Microsoft Research
-  url: https://github.com/olgavrou/
-  image_url: https://github.com/olgavrou.png
-
-joshkyh:
-  name: Joshua Kim
-  title: AI Freelancer at SpectData
-  url: https://github.com/joshkyh/
-  image_url: https://github.com/joshkyh.png
-
-freedeaths:
-  name: Yishen Sun
-  title: Data Scientist at PingCAP LAB
-  url: https://github.com/freedeaths/
-  image_url: https://github.com/freedeaths.png
-
-yifanzeng:
-  name: Yifan Zeng
-  title: PhD student at Oregon State University
-  url: https://xhmy.github.io/
+sonichi:
```

Comment:
It seems there is a problem with CRLF and LR format for file ending with this file.